xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -122,7 +122,7 @@ This constructor assumes that the entire JSON payload is contained in `jsonData`
         <param name="jsonData">The UTF-8 encoded JSON text to process.</param>
         <param name="isFinalBlock">
           <see langword="true" /> if the input sequence contains the entire data to process; <see langword="false" /> only the input span contains partial data with more data to follow.</param>
-        <param name="state">An object that contains the reader state. If this is the first call to the constructor, pass in a default state. Otherwise, capture the state from the previous instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> and pass that back.</param>
+        <param name="state">A structure that contains the reader state. If this is the first call to the constructor, pass in a default state. Otherwise, capture the state from the previous instance of the <see cref="T:System.Text.Json.Utf8JsonReader" /> by calling <see cref="T:System.Text.Json.Utf8JsonReader.CurrentState" /> and pass that back.</param>
```

Comment:
Similarly here, JsonReaderState is a struct, not an class. Changed "object" to "structure".

Reply:
potato vs potato 😄

Reply:
I'll fix this instance.